tools/sof_perf_analyzer.py

```diff
@@ -274,8 +271,7 @@ def analyze_perf_info():
     perf_stats.columns = ['CPU_AVG(MIN)', 'CPU_AVG(AVG)', 'CPU_AVG(MAX)',
                           'CPU_PEAK(MIN)', 'CPU_PEAK(AVG)', 'CPU_PEAK(MAX)']
     perf_stats['PEAK(MAX)/AVG(AVG)'] = perf_stats['CPU_PEAK(MAX)'] / perf_stats['CPU_AVG(AVG)']
-    comp_peak_avg_cycles = perf_info.groupby('COMP_ID')['CPU_PEAK'].mean()
-    perf_stats['MODULE_CPC'] = comp_peak_avg_cycles * CPC_MARGIN
+    perf_stats['MODULE_CPC'] = perf_info.groupby('COMP_ID')['CPU_PEAK'].max()
```

Comment:
it's also arguable that the CPC value needs to be e.g. 1.2 the max measured value to account for unknowns such as cache and IPC.

 CPC = measured_peak_cpu * CPC_MARGIN

Reply:
ok, peak is already much larger than avg(2-5 times), unless all module peak happen in the same time, then the budget should be ok.